samples/c2pa/C2paPlayer/C2paMenu/C2paMenuFunctions.js

```diff
@@ -98,16 +112,43 @@ export let updateC2PAMenu = function (c2paStatus, c2paMenu , isMonolithic , vide
         );
 
         if (c2paItemValue != null) {
+            //formatting for social media links
+            if(c2paItemKey=== 'SOCIAL' ){
+                var socialArray = c2paItemValue.map(function (account) {
+                    var formattedWebsite = providerInfoFromSocialId(account).name
+                    return `<span><a class="url" href=${account}>${formattedWebsite}</a></span>`
```

Comment:
@dkozma For some reason , this link doesn't redirect the user , on hover it shows the right link and if I right click I can open it in a new tab , but I cant click on it. I'm suspecting there might be a click event from the menu component blocking it cc. @spetrangeli @haoliangx

Reply:
I'm guessing you need to put quotes around the href (you probably have spent too much time in Svelte haha)
```suggestion
                    return `<span><a class="url" href="${account}">${formattedWebsite}</a></span>`
```

Reply:
This doesn't fix it for some reason but if I add this `onclick="window.open('${account}')"` it works